mass_editing/models/mass_object.py

```diff
@@ -77,6 +77,7 @@ def unlink(self):
         self.unlink_action()
         return super(MassObject, self).unlink()
 
+    @api.multi
```

Comment:
No need for this. It's the default

Reply:
Travis was complaining about it was missing, at least in previous versions

Reply:
I'm sure it doesn't.